contrib/platform/src/com/sun/jna/platform/win32/WlanApi.java

```diff
@@ -0,0 +1,188 @@
+package com.sun.jna.platform.win32;
```

Comment:
The license header is missing (see for example https://github.com/java-native-access/jna/blob/dc44b985cc188a3fbf3684a9d47a6e2a8229e648/contrib/platform/src/com/sun/jna/platform/win32/Crypt32.java#L1-L23)